stable/database/README.md

```diff
@@ -229,6 +229,12 @@ The following tables list the configurable parameters of the `database` chart an
 | `configFilesPath` | Directory path where `configFiles.*` are found | `/etc/nuodb/` |
 | `configFiles.*` | See below. | `{}` |
 | `podAnnotations` | Annotations to pass through to the SM an TE pods | `nil` |
+| `autoImport.*` | Enable and configure the automatic initialising of the initial database state | `disabled` |
+| `autoImport.source` | The source - typically a URL - of the database copy to import | `""` |
+| `autoImport.credentials` | Authentication for the download of `source` in the form `user`:`password` | '""'|
+| `autoImport.stripLevels` | The number of levels to strip off pathnames when unpacking a TAR file of an archive | `1` |
+| `autoImport.type` | Type of content in `source`. One of `stream` -> exact copy of an archive; or `backupset` -> a NuoDB hotcopy backupset | 'backupset' |
```

Comment:
Why the name "stream" instead of "archive"?

Reply:
This was the name originally chosen. I'm not sure about the reason. Effectively `stream` is a `tar.gz` of NuoDB archive which is ready to be downloaded.